apps/camera/style/controls.css

```diff
@@ -421,4 +421,4 @@
 .controls.mode-video.capture-active .capture-button .center:before {
   border-radius: 3px;
   background-color: white;
-}
+}
```

Comment:
This is an unnecessary change and makes the patch seem larger that it actually is.  Please add the newline back